lua/starfall/libs_sh/entities.lua

```diff
@@ -1133,6 +1133,34 @@ function ents_methods:getSequenceName(id)
 	return getent(self):GetSequenceName(id)
 end
 
+--- Gets various information about the specified animation
+-- @param number id The ID of the animation
+-- @return table Animation info
+function ents_methods:getSequenceInfo(id)
+	local ent = getent(self)
+	checkluatype(id, TYPE_NUMBER)
+	if id < 0 or id > ent:GetSequenceCount() - 1 then SF.Throw("Sequence ID out of bounds", 2) end
+	return getent(self):GetSequenceInfo(id)
```

Comment:
Looks like this function actually returns vectors in `bbmin` and `bbmax` that need to be wrapped.